sale_order_line_date/models/sale_order_line.py

```diff
@@ -27,13 +27,6 @@ def write(self, vals):
                     break
         return super(SaleOrderLine, self).write(vals)
 
-    @api.model
-    def create(self, vals):
-        res = super(SaleOrderLine, self).create(vals)
-        if res.order_id.commitment_date and not res.commitment_date:
-            res.write({"commitment_date": res.order_id.commitment_date})
-        return res
-
```

Comment:
@cor9003 Why did you remove this method?

Reply:
Hello, we were done several testing and this code never had this casuistic res.order_id.commitment_date and not res.commitment_date, for this reason never should be executed.
Checked the test data in the module we saw that never pass through this code neither and codecov also confirmed it.